src/autoComplete/AutoComplete.tsx

```diff
@@ -103,6 +105,7 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
     readOnly: false,
     required: false,
     error: null,
+    shouldDisplayDivider: false,
```

Comment:
This props is not really explicit (to me at least 😬). I can't understand just by reading the name where it will insert the divider. Also in your screenshot I see the divider in the section and overlay cases 🤔

Reply:
You're right, I've added and extra screenshot with the autoComplete field at its initial state with no divider.
The Divider is used to separate search field and result list so when there is not list the divider is not displayed.

In addition, in the current version the Divider doesn't exist, it will be displayed only in the new search form so this variable help us to be retro compatible with current version

Since I have extra works to do with this component to match with specification, I will change the code in next PR to have only one variable that tells I'm in the search web form.